.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
Hi @wessstt  try not to push the .vscode/settings.json file in the future - this is a personal setting and if someone else gets your code it will mess with their vscode settings -You can gitignore this file